code/edgeconsent/src/test/java/com/adobe/marketing/mobile/edge/consent/ConsentExtensionTest.java

```diff
@@ -68,7 +68,9 @@ public class ConsentExtensionTest {
 
 	@Before
 	public void setup() {
-		PowerMockito.mockStatic(MobileCore.class);
+		//TODO
+		Mockito.reset(mockExtensionApi);
+		Mockito.mockStatic(MobileCore.class);
```

Comment:
Getting error with line when running all the tests.

"For com.adobe.marketing.mobile.MobileCore, static mocking is already registered in the current thread

To create a new mock, the existing static mock registration must be deregistered
org.mockito.exceptions.base.MockitoException: 
For com.adobe.marketing.mobile.MobileCore, static mocking is already registered in the current thread"